controllers/bpfman-operator/configmap.go

```diff
@@ -321,6 +322,7 @@ func LoadAndConfigureBpfmanDs(config *corev1.ConfigMap, path string) *appsv1.Dae
 	staticBpfmanDeployment.Spec.Template.ObjectMeta.Annotations["bpfman.io.bpfman.agent.metricaddr"] = bpfmanMetricAddr
 	staticBpfmanDeployment.Name = internal.BpfmanDsName
 	staticBpfmanDeployment.Namespace = config.Namespace
+	staticBpfmanDeployment.Spec.Template.Spec.AutomountServiceAccountToken = ptr.To(true)
```

Comment:
This may be a stupid question, but I'm trying to understand this change.

If we weren't automounting the service account token before, were we manually mounting it or passing it in somehow?  And, If so, should that mechanism be removed?

Reply:
this is a valid question for the manual token creation there should have been 
`oc create token <token-name>` somewhere in the process but I couldn't find that 
so to me this looked like a bug that was waiting to happen when trying to mount secrets volume `/var/run/secrets/kubernetes.io/serviceaccount` for specific serviceAccount